File: ruby/fib.rb
n = 30
repeat_count = 10

def fib(n)
  n < 2 ? n : fib(n - 2) + fib(n - 1)
end

start = Time.now
repeat_count.times { fib(n) }
_end = Time.now

puts ((_end - start) * 1000).to_i

File: c#/fib.cs
using System;
using System.Diagnostics;

class Program {
    const int N = 30;
    const int RepeatCount = 10;

    static int Fib(int n) => n < 2 ? n : Fib(n - 2) + Fib(n - 1);

    static void Main() {
        var sw = new Stopwatch();
        sw.Start();
        for (var i = 0; i < RepeatCount; ++i) Fib(N);
        sw.Stop();
        Console.WriteLine(sw.ElapsedMilliseconds);
    }
}
